src/Orleans.Core/Hosting/GenericHostExtensions.cs

```diff
@@ -22,7 +22,7 @@ public static class OrleansClientGenericHostExtensions
         /// Note that this method should not be used in conjunction with IHostBuilder.UseOrleans, since UseOrleans includes a client automatically.
```

Comment:
Any remarks we need to include around using the `HostBuilderContext` in this delegate?

Reply:
I don't think so, but if you see a nice way to squeeze them in, I'd be happy to